ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -360,6 +364,45 @@ def ocs_install_verification(
         check_fips_enabled()
 
 
+def osd_encryption_verification():
+    """
+    osd_encryption_verification
+
+    Returns:
+        bool: True in case no configure encryption mode.
+    """
+    if not config.ENV_DATA.get("encryption_at_rest"):
+        return True
+    ocs_version = float(config.ENV_DATA['ocs_version'])
+    if ocs_version < 4.6:
+        error_message = "Encryption at REST can be enabled only on OCS >= 4.6!"
+        raise UnsupportedFeatureError(error_message)
```

Comment:
I see you are raising several Exceptions here but missing them in docstring in `Raises:` sections